src/testing.ts

```diff
@@ -25,7 +25,7 @@ export async function run() {
         testsuitesTitle: `VS Code (${version}) Extension Tests: Mocha (${process.platform} ${process.arch})`,
         mochaFile: resultFilePath,
         useFullSuiteTitle: true,
-        rootSuiteTitle: "Tests",
+        rootSuiteTitle: `${version}:`,
```

Comment:
"Tests" was pretty worthless and redudntant, so I figure setting the VS Code version here at least helps to see what may have passed/failed at a glance:

<img width="575" height="206" alt="Image" src="https://github.com/user-attachments/assets/24cb7a9e-5166-4821-bce3-b77049baf891" />